nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Please ensure that you return an Error message to the user, possible solution would be to add this line: `"#{self.class} has not implemented method '#{__method__}'"` after: `NotImplementedError`